apps/src/code-studio/peer_reviews/PeerReviewLinkSection.jsx

```diff
@@ -40,18 +39,6 @@ class PeerReviewLinkSection extends React.Component {
       </ul>
     );
   }
-
-  render() {
-    if (this.props.filterType === 'escalated') {
```

Comment:
Is filtertype being removed because we now want to show all submissions and not just the escalated ones?